py-scripts/throughput_qos.py

```diff
@@ -1130,17 +1212,17 @@ def generate_individual_graph(self, res, report):
                         report.set_table_title(" TOS : Video ")
                         report.build_table_title()
                         video_dataframe = {
-                            " Client Name " : self.sta_list,
```

Comment:
In the future, please submit whitespace changes separate from logic changes. Couple more instances here, but I'll refrain from commenting more.